fasthadd.spec

```diff
@@ -1,10 +1,11 @@
-### RPM external fasthadd 2.3
+### RPM external fasthadd 2.4
 
 #Change the commit hash every time a new version is needed.
-%define commit 905af02d3369428df677c232537da6fca8982ff3
-Source0: https://raw.githubusercontent.com/cms-sw/cmssw/%commit/DQMServices/Components/bin/fastHadd.cc
-Source1: https://raw.githubusercontent.com/cms-sw/cmssw/%commit/DQMServices/Core/src/ROOTFilePB.proto
-Requires: protobuf root
+%define commit 003102d8516b652d80e76d8ca1c4b5dc47c9015b
+%define user iarspider
```

Comment:
@iarspider , I think now we can get the updated files from cmssw repo ... right? can you please open a Pr with change